contracts/interfaces/ITwabRewards.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.6;
+
+import "@pooltogether/v4-core/contracts/interfaces/ITicket.sol";
+
+/**
+ * @title  PoolTogether V4 ITwabRewards
+ * @author PoolTogether Inc Team
+ * @notice TwabRewards contract interface.
+ */
+interface ITwabRewards {
+    /**
+        @notice Struct to keep track of each epoch's settings.
+        @param id Epoch id to keep track of each epoch
+        @param startTimestamp Timestamp at which the epoch starts
+        @param duration Duration of one epoch in seconds
+     */
+    struct Epoch {
+        uint32 id;
+        uint32 startTimestamp;
+        uint32 duration;
+    }
+
+    /**
+        @notice Struct to keep track of each promotion's settings.
+        @param id Promotion id to keep track of each promotion
+        @param creator Addresss of the promotion creator
+        @param ticket Prize Pool ticket address for which the promotion has been created
+        @param token Address of the token to be distributed as reward
+        @param tokensPerEpoch Number of tokens to be distributed per epoch
+        @param startTimestamp Timestamp at which the promotion starts
+        @param epochDuration Duration of one epoch in seconds
+        @param numberOfEpochs Number of epochs the promotion will last for
+     */
+    struct Promotion {
+        uint32 id;
+        address creator;
+        address ticket;
+        address token;
+        uint256 tokensPerEpoch;
+        uint32 startTimestamp;
+        uint32 epochDuration;
+        uint256 numberOfEpochs;
+    }
+
+    /**
+        @notice Parameters needed to create a promotion.
+        @param token Address of the token to be distributed
+        @param tokensPerEpoch Number of tokens to be distributed per epoch
+        @param startTimestamp Timestamp at which the promotion starts
+        @param epochDuration Duration of one epoch in seconds
+        @param numberOfEpochs Number of epochs the promotion will last for
+     */
+    struct PromotionParameters {
+        address token;
+        uint256 tokensPerEpoch;
+        uint32 startTimestamp;
+        uint32 epochDuration;
+        uint256 numberOfEpochs;
+    }
```

Comment:
I don't think we need a struct for this.

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/8/commits/bc5f7b27d921335a65cc47925270cb37dcca3213